runelite-client/src/main/java/net/runelite/client/plugins/timers/GameTimer.java

```diff
@@ -92,28 +99,69 @@
 		this.duration = Duration.of(time, unit);
 	}
 
+	GameTimer(int itemId, int quantity, String description, Integer graphicId, long time, ChronoUnit unit)
+	{
+		this("", description, graphicId, time, unit);
+		this.itemId = itemId;
+		this.itemQuantity = quantity;
+	}
+
+	GameTimer(int spriteId, String description, Integer graphicId, long time, ChronoUnit unit)
+	{
+		this("", description, graphicId, time, unit);
+		this.spriteId = spriteId;
+	}
+
 	GameTimer(String imageResource, String description, long time, ChronoUnit unit)
 	{
 		this(imageResource, description, null, time, unit);
 	}
 
-	public BufferedImage getImage()
+	GameTimer(int itemId, int quantity, String description, long time, ChronoUnit unit)
```

Comment:
See my previous comment. If you need to differentiate i would use some enum or something probably

Reply:
It's enums all the way down!